docs/web-apps/automated-testing/_partials/_advanced-cypress.md

```diff
@@ -32,4 +32,8 @@ When running Cypress test along with Cucumber, you can use tags to select which
 
 ```
 saucectl run --env CYPRESS_TAGS="(@smoke or @ui) and (not @slow)"
-```
\ No newline at end of file
+```
+
+## Cypress Component Test
+
+Cypress Component Test is still in Beta. Due to an issue from Cypress, `saucectl` doesn't support Cypress Component Test currently.
```

Comment:
```suggestion
Component Testing is in beta and due to an issue from Cypress, `saucectl` doesn't currently support Component Testing.
```

Reply:
That's an interesting topic. The two issues that I found was solved, but it is still stuck in finding spec files. I've searched in cypress issues but currently no luck. I'm considering filing one.